deployments/AKS/kubearmor.yaml

```diff
@@ -304,23 +304,6 @@ spec:
         kubearmor-app: kubearmor-policy-manager
     spec:
       containers:
-      - args:
-        - --secure-listen-address=0.0.0.0:8443
-        - --upstream=http://127.0.0.1:8080/
-        - --logtostderr=true
-        - --v=10
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
```

Comment:
@rksharma95 have we tried using a newer image of kube-rbac-proxy instead of v0.5.0? I think they do compile it for ARM. :thinking: 
Ref - https://console.cloud.google.com/gcr/images/kubebuilder/global/kube-rbac-proxy

Reply:
cc : @achrefbensaad

Reply:
> @rksharma95 have we tried using a newer image of kube-rbac-proxy instead of v0.5.0? I think they do compile it for ARM. thinking Ref - https://console.cloud.google.com/gcr/images/kubebuilder/global/kube-rbac-proxy

yes it's working fine with newer images i tested it with v0.8.0